index.html

```diff
@@ -1590,6 +1592,10 @@ <h3>
         <ol>
           <li>Let |promise:Promise| be [=a new promise=].
           </li>
+          <li>If [=this=].{{HIDDevice/[[state]]}} is either `"forgotten"` or
+          `"forgetting"`, then [=reject=] |promise| with a
```

Comment:
Currently close() never rejects and I think it would be nice to keep it that way if we can. If close() is called on a "forgotten" or "forgetting" device let's resolve instead of rejecting so developers don't have to handle it like an error.

Reply:
`close()` actually rejects if `open()` is not finished yet in Chromium implementation.

```js
device.open();
device.close();
// Promise {<rejected>: DOMException: An operation that changes the device state is in progress.}
```

Resolving `close()` while the device is actually not accessible seems weird to me.